ci/build.yml

```diff
@@ -479,7 +501,8 @@ legacy emu btconly debug build:
 legacy emu btconly debug asan build:
   stage: build
   <<: *gitlab_caching
-  needs: []
+  needs:
+    - core unix frozen debug build  # core tests have priority
```

Comment:
this changeset will shutdown the whole pipeline in case core unix frozen debug build is broken. let's figure out something else

Reply:
It would crash the whole pipeline, but when `frozen debug build` is broken, then the majority of the tests will not run anyway.

I am not against reverting this, it is really not so helpful now.

Reply:
please revert it then.

you're right about the "majority" of tests, but it's going to be annoying especially for commits that do something to legacy build while the core is not-yet-updated (or similar with btc-only variant)

Reply:
Reverted in [d8b5118](https://github.com/trezor/trezor-firmware/pull/2857/commits/d8b5118ec6e4b3680548f5641afe490ea60071ff)